src/node.cc

```diff
@@ -2753,8 +2753,11 @@ static void EnvGetter(Local<Name> property,
     return info.GetReturnValue().SetUndefined();
   }
   node::Utf8Value key(isolate, property);
-  char buffer[32767];
-  int ret = uv_os_getenv(*key, buffer, arraysize(buffer));
+  // On POSIX there is not explicitly defined size limit, but on Windows
+  // environment variables have a maximum size limit of 2**15 - 1.
+  char buffer[32768];
+  size_t buf_size = arraysize(buffer);
```

Comment:
Still `arraysize()` 😄

Reply:
I thought I had woken up. Apparently not, so I need more coffee.